samples/sample_common/src/vaapi_utils_drm.cpp

```diff
@@ -82,13 +82,34 @@ int open_intel_adapter(int type)
     return -1;
 }
 
-DRMLibVA::DRMLibVA(int type)
+int open_intel_adapter(int type, const char * devicePath)
+{
+    char driverName[MFX_DRM_DRIVER_NAME_LEN + 1] = {};
+
+    if(strcmp(devicePath,"") == 0)
```

Comment:
why default is `""`? maybe `nullptr` will be simpler?

Reply:
+1 here

Reply:
@mgonchar @dvrogozh if we use char/string, anyway we send empty string here:
```
CreateVAAPIDevice(params.libvaBackend, InputParams.strDevicePath.c_str())
```
So we must check this condition...